stackit/internal/services/iaas/networkarea/datasource.go

```diff
@@ -0,0 +1,254 @@
+func (d *networkAreaDataSource) Schema(_ context.Context, _ datasource.SchemaRequest, resp *datasource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		Description:         "Network area resource schema. Must have a `region` specified in the provider configuration.",
+		MarkdownDescription: features.AddBetaDescription("Network area datasource schema. Must have a `region` specified in the provider configuration."),
+		Attributes: map[string]schema.Attribute{
+			"id": schema.StringAttribute{
+				Description: "Terraform's internal resource ID. It is structured as \"`organization_id`,`network_area_id`\".",
+				Computed:    true,
+			},
+			"organization_id": schema.StringAttribute{
+				Description: "STACKIT organization ID to which the network area is associated.",
+				Required:    true,
+				Validators: []validator.String{
+					validate.UUID(),
+					validate.NoSeparator(),
+				},
+			},
+			"network_area_id": schema.StringAttribute{
+				Description: "The network area ID.",
+				Required:    true,
+				Validators: []validator.String{
+					validate.UUID(),
+					validate.NoSeparator(),
+				},
+			},
+			"name": schema.StringAttribute{
+				Description: "The name of the network area.",
+				Computed:    true,
+				Validators: []validator.String{
+					stringvalidator.LengthAtLeast(1),
+					stringvalidator.LengthAtMost(63),
+				},
+			},
+			"project_count": schema.Int64Attribute{
+				Description: "The amount of projects currently referencing this area.",
+				Computed:    true,
+				Validators: []validator.Int64{
+					int64validator.AtLeast(0),
+				},
+			},
+			"default_nameservers": schema.ListAttribute{
+				Description: "List of DNS Servers/Nameservers.",
+				Computed:    true,
+				ElementType: types.StringType,
+			},
+			"network_ranges": schema.ListNestedAttribute{
+				Description: "List of Network ranges.",
+				Computed:    true,
+				Validators: []validator.List{
+					listvalidator.SizeAtLeast(1),
+					listvalidator.SizeAtMost(64),
+				},
+				NestedObject: schema.NestedAttributeObject{
+					Attributes: map[string]schema.Attribute{
+						"network_range_id": schema.StringAttribute{
+							Computed: true,
+							Validators: []validator.String{
+								validate.UUID(),
+								validate.NoSeparator(),
+							},
+						},
+						"prefix": schema.StringAttribute{
+							Computed: true,
+						},
+					},
+				},
+			},
+			"routes": schema.ListNestedAttribute{
+				Description: "List of routes.",
+				Computed:    true,
+				NestedObject: schema.NestedAttributeObject{
+					Attributes: map[string]schema.Attribute{
+						"route_id": schema.StringAttribute{
+							Computed: true,
+							Validators: []validator.String{
+								validate.UUID(),
+								validate.NoSeparator(),
+							},
+						},
+						"prefix": schema.StringAttribute{
+							Computed: true,
+						},
+						"next_hop": schema.StringAttribute{
+							Computed: true,
+						},
+					},
+				},
+			},
```

Comment:
Won't this fail because the `routes` fields is not yet in the model? (because it's part of a follow-up story, right?)

Reply:
Even if not, since this is not populated yet, I think we could remove this from the schema and add it in that story only